coordinator/migrations/2023-07-19-055143_closed_positions/up.sql

```diff
@@ -0,0 +1,10 @@
+-- Your SQL goes here
+-- Note that the `IF NOT EXISTS` is essential because there is no `down` migration for removing this value because it is not really feasible to remove enum values!
+-- In order to allow re-running this migration we thus have to make sure to only add the value if it does not exist yet.
+ALTER TYPE "PositionState_Type"
+ADD
+    VALUE IF NOT EXISTS 'Closed';
+ALTER TABLE
```

Comment:
🙃 Maybe not 100% related to your change, but it would be great if we'd also save the closing price on the position and not just the pnl.

Reply:
Yeah... I thought about doing that, but it would be yet another piece of information we would save at a different point in time. I guess it will be useful to have for validation. I will see to add it tomorrow :)